math.java

```diff
@@ -0,0 +1,5 @@
+public class math {
+    public static int add(int num1, int num2) {
+        return num1 + num2;
+    }
```

Comment:
you should write some comments
```suggestion
// Returns the sum of two numbers
public static int add(int num1, int num2) {
    // Add two numbers together
    return num1 + num2;
}
```